dvc/utils/__init__.py

```diff
@@ -183,16 +183,26 @@ def remove(path):
             raise
 
 
-def to_chunks(l, jobs):
-    n = int(math.ceil(len(l) / jobs))
+def to_chunks(l, num_chunks=None, chunk_size=None):
+    if (num_chunks and chunk_size) or (not num_chunks and not chunk_size):
+        from dvc.exceptions import DvcException
 
-    if len(l) == 1:
-        return [l]
+        raise DvcException(
+            "One and only one of 'num_chunks', 'chunk_size' " "must be defined"
+        )
+
+    if num_chunks:
```

Comment:
Let's go the other way around, so we have less code in the branch. Something like:
```
if chunk_size:
    return [l[i : i + chunk_size] for i in range(0, len(l), chunk_size)]

# and now process `num_chunks` case
```